<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogFloodAR</class>
 <widget class="QDialog" name="dialogFloodAR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FloodAR tools</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_ASC_cut">
      <attribute name="title">
       <string>ASC cut</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,1,0">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Tiles size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_tileSize_x">
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_tileSize_y">
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>m</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Source file:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_ASCcut_src"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_ASCcut_in">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QRadioButton" name="radioBtn_ASCCut_terrain">
            <property name="text">
             <string>Terrain</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioBtn_ASCCut_water">
            <property name="text">
             <string>Water</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ASCcut_exec">
         <property name="text">
          <string>Launch ASC cut</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_tex_cut">
      <attribute name="title">
       <string>Texture cut</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1,0,1,0">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Tiles size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_txTileSize_x">
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_txTileSize_y">
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>m</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Source file:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_texCut_src"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_texCut_in">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_texCut_exec">
         <property name="text">
          <string>Launch Texture cut</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_shpcut">
      <attribute name="title">
       <string>SHP cut</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="0,1,0,1,0">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Tiles size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_shpTileSize_x">
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_shpTileSize_y">
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>m</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Source file:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_shpCut_src"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_shpCut_in">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_shpCut_exec">
         <property name="text">
          <string>Launch SHP cut</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>ASC to CityGML</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Terrain</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_ASCtoTerrain_exec">
            <property name="text">
             <string>Convert ASC files to CityGML Terrain</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Water</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="1" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>Starting Date</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>Water height precision:  </string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="dbSpinBox_ASCtoWater_prec">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="decimals">
                  <number>3</number>
                 </property>
                 <property name="minimum">
                  <double>0.001000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.100000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="label_9">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>m.</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDateTimeEdit" name="dtEdit_creationDate">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="btn_ASCtoWater_exec">
              <property name="text">
               <string>Convert ASC files to CityGML Water</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_ShpExt">
      <attribute name="title">
       <string>SHP Extrusion</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="btn_ShpExt_buildAABB">
         <property name="text">
          <string>Build Bounding Boxes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ShpExt_exec">
         <property name="text">
          <string>Convert SHP to CityGML Building</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Working directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_wkingDir"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_wkingDir">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="btn_exit">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
